docs/source/cuda_environment_variables.rst

```diff
@@ -0,0 +1,41 @@
+.. _cuda_environment_variables:
+
+CUDA Environment Variables
```

Comment:
How does this page align with this: https://pytorch.org/docs/stable/notes/cuda.html#memory-management ? Should there be some sort of cross-linking?

Reply:
great callout! I wll add a link for these memory management related env variables